### DISCUSSION TITLE
GitHub pro

### DISCUSSION THREAD

**Discussion by Mukeshmikey** [community/community#112436] [GitHub Education]
I have applied for GitHub student starter pack and submitted all proofs on March 12th and it says approved. So, when will I able to get those pro benefits. Please help me .

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Mukeshmikey! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
